codebuild_specs/scripts-windows/run-e2e-windows.sh

```diff
@@ -8,7 +8,7 @@ export MSYS2_ARG_CONV_EXCL="*"
 
 export AMPLIFY_DIR=$CODEBUILD_SRC_DIR\\out
 export AMPLIFY_PATH=$CODEBUILD_SRC_DIR\\out\\amplify.exe
-export NODE_OPTIONS=--max-old-space-size=4096
+export NODE_OPTIONS=--max-old-space-size=5120
```

Comment:
Why not using an even-bigger number?

Reply:
because we run multiple test workers on machine, therefore increasing in steps as needed, so the chance they'll consume all available memory on VM is less.